timemachine/cpp/src/local_md_potentials.cu

```diff
@@ -0,0 +1,190 @@
+#include "constants.hpp"
+#include "gpu_utils.cuh"
+#include "kernel_utils.cuh"
+#include "kernels/k_flat_bottom_bond.cuh"
+#include "kernels/k_indices.cuh"
+#include "kernels/k_local_md.cuh"
+#include "local_md_potentials.hpp"
+#include "math_utils.cuh"
+#include <cub/cub.cuh>
+#include <random>
+#include <vector>
+
+namespace timemachine {
+
+// Struct to as a CUB <= operation
```

Comment:
<= or < ?

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1004/commits/57f21ba6cf5a201ba2f7a99f3580474e81a82570